scripts/compile.mjs

```diff
@@ -56,3 +56,12 @@ await fs.promises.writeFile(
     <script src="./hglib.min.js"></script>
   `)
 );
+
+// =========================================
+//       Copy test helpers to dist
+// =========================================
+
+await fs.promises.copyFile(
```

Comment:
We just copy this file to dist. We shouldn't need to bundle since `higlass` is already bundled by us